src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -416,6 +416,60 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         return this;
     }
 
+
+
+
+    public WebElement swipeTo(SwipeElementDirection s, By by, int attempts){
+
+        int i;
+
+        if (isPresentWait(by)){
+            return driver.findElement(by);
+        }
+        else {
```

Comment:
`else` should be on the same line as `{` one line above.

Reply:
done